napari/utils/colormaps/colormap.py

```diff
@@ -275,11 +282,21 @@ def _background_as_minimum_dtype(self, dtype: np.dtype) -> int:
         int
             The background converted.
         """
-        return int(
-            _cast_labels_data_to_texture_dtype_auto(
-                dtype.type(self.background_value), self
-            )
-        )
+        return int(self._map_to_gpu(dtype.type(self.background_value)))
+
+    @overload
+    def _map_to_gpu(self, values: np.ndarray) -> np.ndarray:
+        ...
+
+    @overload
+    def _map_to_gpu(self, values: np.integer) -> np.integer:
+        ...
+
+    def _map_to_gpu(
+        self, values: Union[np.ndarray, np.integer]
+    ) -> Union[np.ndarray, np.integer]:
+        """Map input values to values for send to GPU."""
```

Comment:
```suggestion
    @overload
    def _data_to_texture_dtype(self, values: np.ndarray) -> np.ndarray:
        ...

    @overload
    def _data_to_texture_dtype(self, values: np.integer) -> np.integer:
        ...

    def _data_to_texture_dtype(
        self, values: Union[np.ndarray, np.integer]
    ) -> Union[np.ndarray, np.integer]:
        """Map input values to a (possibly reduced) dtype to set a GPU texture."""
```

... Typing question: since the overload is defined in the base class, is it also needed here?

Reply:
mypy was complaining about lack of overload...